samples/client_subscribe.py

```diff
@@ -36,8 +36,8 @@ async def uptime_coro():
         await C.unsubscribe(["$SYS/broker/uptime", "$SYS/broker/load/#"])
         logger.info("UnSubscribed")
         await C.disconnect()
-    except ClientException as ce:
-        logger.error("Client exception: %s" % ce)
+    except ClientException:
+        logger.error("Client exception", exc_info=True)
```

Comment:
```suggestion
        logger.exception("Client exception")
```

As you suggested in #105 the `exception` shortcut is nice :+1: This could be an opportunity to move to it.